core/css.ts

```diff
@@ -119,7 +118,7 @@ let content = `
   box-shadow: 0 0 3px 1px rgba(0,0,0,.3);
 }
 
-.blocklyDropDownDiv.blocklyFocused {
```

Comment:
There is another reference to `.blocklyFocused` in this file, could you remove it as well?